src/main/java/com/dev/cinema/Main.java

```diff
@@ -16,9 +16,11 @@
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
+import java.util.logging.Logger;
```

Comment:
wrong class imported in all the projects

also, you are missing `log4j.properties` configuration file